seeds/seeds.js

```diff
@@ -1,22 +1,71 @@
-const seedFile = require('knex-seed-file');
+const Promise = require("bluebird");
+
+require("colors");
 
 exports.seed = function(knex, Promise) {
-  return Promise.join(
-    // Deletes ALL existing entries
-    knex('orgs').del(), 
+  console.log("Running seeds.js".yellow)
+  console.log("Deleting all tables".yellow)
+  return knex.raw(
+    `DROP SCHEMA public CASCADE;
+    CREATE SCHEMA public;`
+  ).then(() => {
```

Comment:
Just dropping all the tables, a little heavy-handed, want to make sure this isn't ever run on anything important, maybe add a check that we're on a testing database